lib/rubyflake.rb

```diff
@@ -27,14 +27,14 @@ def generate
   # Extract the time from an ID.
   #
   def time(id)
-    Time.at(((id >> Rubyflake::FLAKE_TIMESTAMP_SHIFT) + @epoch) / 1000)
+    Time.at(((id >> Rubyflake::FLAKE_TIMESTAMP_SHIFT) / 1000) + @epoch)
```

Comment:
Looks like tagging you worked. :-) Consider adding Joallard as a contributor, we could get this show on the road.